HIRS_Utils/src/test/java/hirs/validation/SupplyChainCredentialValidatorTest.java

```diff
@@ -1992,6 +1995,211 @@ public final void testValPlatCredentialAttributesV2p0RequiredComponentNoSerialOr
                 SupplyChainCredentialValidator.PLATFORM_ATTRIBUTES_VALID);
     }
 
+    /**
+     * Tests that SupplyChainCredentialValidator passes with a base and delta certificate where
+     * the base serial number and delta holder serial number match.
+     * @throws java.io.IOException Reading file for the certificates
+     * @throws java.net.URISyntaxException when loading certificates bytes
+     */
+    @Test
+    public final void testValidateDeltaPlatformCredentialAttributes()
+            throws IOException, URISyntaxException {
+        DeviceInfoReport deviceInfoReport = setupDeviceInfoReportWithComponents(
+                SAMPLE_PACCOR_OUTPUT_TXT);
+
+        PlatformCredential base = mock(PlatformCredential.class);
```

Comment:
There's a lot of repeated code between the two test methods, can you consolidate under a @Before method for brevity?

Reply:
At this time, no changes will be made.